lib/levels.js

```diff
@@ -119,7 +113,36 @@ function getLevel (level) {
 function isLevelEnabled (logLevel) {
   const { values } = this.levels
   const logLevelVal = values[logLevel]
-  return logLevelVal !== undefined && (logLevelVal >= this[levelValSym])
+  return logLevelVal !== undefined && this[levelCompSym](logLevelVal, this[levelValSym])
+}
+
+/**
+ *
+ * @description compare `current` level with `expected` to make sure `current` level is enabled or not
+ * @param {SORTING_ORDER} direction
+ * @param {number} current
+ * @param {number} expected
+ * @returns {boolean}
+ */
+function compareLevel (direction, current, expected) {
+  if (direction === SORTING_ORDER.DESC) {
+    return current <= expected
+  }
+
+  return current >= expected
+}
+
+/**
+ * @description create a level comparison function based on `levelComparison`
```

Comment:
This description does not help me understand the purpose of this function. What is the purpose of creating such a function? What use cases is the resulting function intended for?

Please also omit the explicit `@description` tag in favor of an opening text block.

Reply:
Done